set.mm

```diff
@@ -801861,525 +801980,803 @@ Graph theory (extension)
 
 $(
 -.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-
-  Isomorphic graphs
+  Isomorphisms of graphs
 -.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-
 
-  In the following, a general definition of the isomorphy relation for graphs
-  and specializations for simple hypergraphs ( ~ isomushgr ) and simple
-  pseudographs ( ~ isomuspgr ) are provided.  The latter corresponds to the
-  definition in [Bollobas] p. 3).  It is shown that the isomorphy relation for
-  graphs is an equivalence relation ( ~ isomgrref , ~ isomgrsym , ~ isomgrtr ).
-  Fianlly, isomorphic graphs with different representations are studied
-  ( ~ strisomgrop , ~ ushrisomgr ).
+  This section is about isomorphisms of graphs, whereby the term "isomorphism"
+  is used in both of its meanings (according to the Meriam-Webster dictionary,
+  see ~ https://www.merriam-webster.com/dictionary/isomorphism ): "1: the
+  quality or state of being isomorphic." and "2: a one-to-one correspondence
+  between two mathematical sets".
+
+  At first, an operation ` GraphIso `is defined (see ~ df-grim ) which provides
+  the graph isomorhisms (as "one-to-one correspondence") between two given
```

Comment:
```suggestion
  the graph isomorphisms (as "one-to-one correspondence") between two given
```

Reply:
There is the same typo at a second place (copy&paste...).